src/parser.cr

```diff
@@ -233,5 +261,24 @@ module Mint
 
       result
     end
+
+    # Getting substrings out of the original string
+    # ----------------------------------------------------------------------------
+
+    def substring(from, to)
```

Comment:
```suggestion
    # Gets substring out of the original string
    def substring(from, to)
```

Reply:
Oh, I see. I was just following this style of comments that are in this file. I'll change it!